docs/tutorials/docker.rst

```diff
@@ -1,12 +1,13 @@
 Run using docker-compose
 =========================================
-Execute the following command the deployment will be started using docker-compose. This will build the dockerimage
+Execute the following commands to start the deployment using docker-compose. This will build the docker image
 containing the application and creates all required containers including the database and the message queue.
 
 Note that you might need to authorize with an Access Token in order to access the qunicorn package on GitHub.
 In this Case the following steps need to be performed.
 
-1. Create a personal access token on Github (Account --> Settings --> Developer Settings --> Personal acces tokens)
+1. Create a personal access token on Github:
+    Navigate to Account → Settings → Developer Settings → Personal access tokens
```

Comment:
There is actually a specific role for this use case: <https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-menuselection>

Reply:
Added menu selection https://github.com/qunicorn/qunicorn-core/pull/34/commits/db1fd98277e42d1d62a7604bf4e0f6d2c12433e0